rhc-ose-ansible/roles/openshift-provision/tasks/prerequisites.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+  
+- name: "Install required packages"
+  yum:
```

Comment:
Just for clarification, this should be **{{ ansible_pkg_mgr }}** right (underscores not dashes)? Aside from changing these tasks to use this action and variable, do I need to do anything else? Also, I can't find any specific documentation on this so I'm not sure how to do the equivalent **yum clean all**. It seems this may be an internal ansible value from gathered facts, so just calls the module. The yum module has **update_cache** which seems close but this doesn't exist for **dnf** so not sure what to do here.